.github/workflows/example-project.yml

```diff
@@ -80,26 +80,42 @@ jobs:
         run: |
           make
 
+  example-project-msys2:
+
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: msys2 {0}
+
+    steps:
+      - name: Clone Git repository
+        uses: actions/checkout@v2
+
       - name: Install MSYS2
-        if: startsWith(matrix.os, 'windows') && matrix.toolchain-suffix == '-gnu'
         uses: msys2/setup-msys2@v2
         with:
-          release: false
-          update: true
           install: >-
-            base-devel
+            make
+            mingw-w64-x86_64-pkg-config
+            mingw-w64-x86_64-rust
+      #      mingw-w64-x86_64-cargo-c
 
-      - name: Copy installed files
-        if: startsWith(matrix.os, 'windows') && matrix.toolchain-suffix == '-gnu'
+      # NB: cargo-c is available from the package mingw-w64-x86_64-cargo-c
+      # (https://packages.msys2.org/base/mingw-w64-cargo-c),
+      # which is the recommended way to use cargo-c on MSYS2.
+      # For testing purposes, we compile and install the latest version from the Git repo
+      # (which takes much longer than installing the MSYS2 package!):
+
+      - name: Install cargo-c applet
+        run: |
+          cargo install --path .
+
+      - name: Install C API for example project
         working-directory: example-project
         run: |
-          # NB: --prefix doesn't seem to matter on Windows
-          cp -r temp/usr/local/{lib,bin,include} /mingw64/
-        shell: msys2 {0}
+          cargo cinstall --verbose --release --prefix /mingw64 --target x86_64-pc-windows-gnu
```

Comment:
I guess cargo has the msvc toolchain as default.

Reply:
Yeah, looks like it.

I was hoping to get the right target automatically, since I'm using `cargo` from MSYS2, which should know that it uses the GNU toolchain.

Strangely, when I'm using the MSYS2 package for `cargo-c` (instead of compiling it on-the-fly), the `--target` flag is *not* necessary (see https://github.com/mgeier/cargo-c/runs/4754623622)!

So why does the MSYS2 package of `cargo-c` do the right thing?

But, as I said, I don't think this should be a blocker for now, we can try to figure this out later.

Reply:
I've created a separate issue as a reminder: #244.

Reply:
For completeness, I've also created one about the `--prefix`: #245